tests/test_model_v2.py

```diff
@@ -596,6 +596,150 @@ def test_traits_datetime():
     assert str(e_info.value).startswith('Cannot decode datetime from value xyz')
 
 
+def test_traits_datetimeoffset():
```

Comment:
Rather then one big test with comments like " # 1. direction Python -> OData" it would be better to simply just split to multiple isolated tests anyway..

Reply:
Will do

Reply:
If we have some tests like that, than shame on us as well and we should rewrite that to shorter, isolated test cases :)